calico_containers/tests/fv/__init__.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sh
+from sh import docker
+from docker_host import DockerHost
+
+
+def setup_package():
+    """
+    Sets up docker images and host containers for running the STs.
+    """
+    containers = docker.ps("-qa").split()
```

Comment:
This chunk is repeated here and in setup and teardown.  Just add another helper method to the DockerHost class.